src/test/java/edu/harvard/iq/dataverse/DatasetFieldServiceBeanDansTest.java

```diff
@@ -0,0 +1,175 @@
+import java.util.List;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DatasetFieldServiceBeanDansTest {
+
+    private DatasetFieldServiceBean datasetFieldServiceBean;
+
+    static String getCvocJson(String pathToJsonFile) throws IOException {
+        final File datasetVersionJson = new File(pathToJsonFile);
+        return new String(Files.readAllBytes(Paths.get(datasetVersionJson.getAbsolutePath())));
+    }
+
+    @BeforeEach
+    void setUp() {
+      this.datasetFieldServiceBean = Mockito.spy(new DatasetFieldServiceBean());
+
+//        Logger rootLogger = Logger.getLogger(DatasetFieldServiceBean.class.getCanonicalName());
+//        rootLogger.setLevel(Level.FINE);
+//
+//        // Ensure all handlers respect the FINE level
+//        for (var handler : rootLogger.getHandlers()) {
+//            handler.setLevel(Level.FINE);
+//        }
+//
+//        // Add a ConsoleHandler if none exists
+//        if (rootLogger.getHandlers().length == 0) {
+//            ConsoleHandler consoleHandler = new ConsoleHandler();
+//            rootLogger.addHandler(consoleHandler);
+//        }
+    }
+
+    @AfterEach
+    void tearDown() {
+      this.datasetFieldServiceBean = null;
+    }
+
+    @Test
+    void interceptedResultForAudience() { // TODO remove when assertTermNameValues succeeds in another test
+        JsonObject result = Json.createReader(new StringReader("""
+            {
+               "@id": "https://www.narcis.nl/classification/D18100",
+               "termName": [
+                 {
+                   "lang": "nl",
+                   "value": "Exploitatie en beheer van het fysieke milieu"
+                 },
+                 {
+                   "lang": "en",
+                   "value": "Exploitation and management of the physical environment"
+                 }
+               ],
+               "vocabularyUri": "https://www.narcis.nl/classification/"
+            }
+            """)).readObject();
+        List<String> expectedValues = List.of(
+            "Exploitation and management of the physical environment",
+            "Exploitatie en beheer van het fysieke milieu"
+        );
+        assertThat(result.getString("@id")).isEqualTo("https://www.narcis.nl/classification/D18100");
+        assertTermNameValues(result, expectedValues);
+        assertThat(result.keySet()).containsExactlyInAnyOrder("@id", "termName", "vocabularyUri");
+        // TODO shouldn't we also get the vocabularyName?
+    }
+
+    @Test
+    void getIndexableStringsForAudience() throws Exception {
+        String termURI = "https://www.narcis.nl/classification/D18100";
+        JsonObject cvocEntry = createMocks("dansAudience", termURI, "audience.json");
+        JsonObject readObject = readObject("src/test/resources/json/cvoc-dans-value/audience.json");
```

Comment:
audience.json doesn't contain an entry for https://www.narcis.nl/classification/D18100